charts/nginx-ingress/values.yaml

```diff
@@ -343,8 +343,8 @@ controller:
   ## Enable OIDC policies.
   enableOIDC: false
 
-  ## Include year in log header. This parameter will be removed in release 3.7 and the year will be included by default.
-  includeYear: false
+  # Include year in log header. This parameter will be removed in release 3.7 and the year will be included by default.
+  includeYear: true
```

Comment:
We do want to remove the value from this helm values file. Essentially it will not be a user option.